ls1intum/Artemis
# [Bugfix] Small fixes for inline feedback component

### Checklist

### Motivation and Context


This PR fixes some small issues found during a testing session.
### Description

- When there is no score set, the tutor cannot save the inline feedback (also sets the background to yellow )
- When you click now on the result label, it shows the correct line number
- Fix wording

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Create a programming exercise, participate and assess
4. Try out the to delete the score in th input field and check whether you can click on save or not
5. Check result label dialogs, should now list the same row as displayed in the gutter
6. Delete a unreferenced feedback element and check for the delete dialog.
7. Play around and see if you can break something

### Test Coverage

Nothing changed

### Screenshots



Not able to click on save when the score is not defined. Background is then also yellow, when score not defined.
<img width="1000" alt="image" src="https://user-images.githubusercontent.com/41108487/97515265-003ecc80-1991-11eb-98b1-4d4c92282a4f.png">

Result label shows the correct line number now:
<img width="1245" alt="image" src="https://user-images.githubusercontent.com/41108487/97515337-26fd0300-1991-11eb-9d1c-0ed553ea61f1.png">

Delete a unreferenced feedback show the following:
<img width="1146" alt="image" src="https://user-images.githubusercontent.com/41108487/97515378-37ad7900-1991-11eb-9f82-41d545b1eabb.png">
